webapp/sass/components/_modal.scss

```diff
@@ -12,7 +12,6 @@
 
     .modal-body {
         max-height: calc(90vh - 62px);
-        overflow: auto;
```

Comment:
Does this still work for other modals like account settings or the two modals in the security settings? It seems like those might expect this to be set